rasa/core/tracker_store.py

```diff
@@ -462,16 +521,81 @@ async def retrieve(self, sender_id: Text) -> Optional[DialogueStateTracker]:
         Returns:
             Tracker containing events from the latest conversation sessions.
         """
+        return await self._retrieve(sender_id, fetch_all_sessions=False)
+
+    async def retrieve_full_tracker(
```

Comment:
Every derived class will inherit implementation of the `retrieve_full_tracker` from TrackerStore class.
The only method they should re-implement is the `retrieve` method.